extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -397,6 +397,8 @@ export class AmpAndroidAppBanner extends AbstractAppBanner {
  */
 function handleDismiss(state) {
   hideBanner(state);
+  // Restore focus to document after dismiss for screen readers.
+  state.element.ownerDocument.body./*REVIEW*/focus();
```

Comment:
Talked to Dima, my assumption was wrong, I think your suggestion of checking `document.activeElement` might work. Will test it out.